src/screens/HomeScreen.jsx

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import Categories from '../components/Categories/Categories';
+import Navbar from '../components/Nav/Navbar';
+
+const HomeScreen = () => {
+    const [category, setCategory] = useState([]);
+    const [error, setError] = useState("");
```

Comment:
This is really good that you added a state for error. You can take it to next lavel by using a loading state 🤠

Reply:
Ok, Ill do it.